debian/control

```diff
@@ -1021,6 +1028,21 @@ Description: Python 3 libraries for the Ceph libcephfs library
  .
  This package contains the debugging symbols for python3-cephfs.
 
+Package: python-ceph-argparse
+Architecture: linux-any
+Section: python
+Depends: ${misc:Depends},
+         ${python:Depends},
+Replaces: ceph-common (<< 14.0.0)
+Breaks: ceph-common (<< 14.0.0)
```

Comment:
looks like this might be causing failure such as (from http://qa-proxy.ceph.com/teuthology/yuriw-2019-05-09_15:50:59-rgw-wip-yuri4-testing-2019-05-08-2017-mimic-distro-basic-smithi/3942466/teuthology.log)

```
019-05-09T16:56:01.598 INFO:teuthology.orchestra.run.smithi089.stdout:The following packages have unmet dependencies:
2019-05-09T16:56:01.638 INFO:teuthology.orchestra.run.smithi089.stdout: ceph-common : Depends: python-cephfs (= 13.2.5-331-gd86b928-1bionic) but it is not going to be installed
2019-05-09T16:56:01.639 INFO:teuthology.orchestra.run.smithi089.stdout:               Depends: python-ceph-argparse (= 13.2.5-331-gd86b928-1bionic) but it is not going to be installed
2019-05-09T16:56:01.641 INFO:teuthology.orchestra.run.smithi089.stdout: python-ceph : Depends: python-cephfs (= 13.2.5-331-gd86b928-1bionic) but it is not going to be installed
2019-05-09T16:56:01.657 INFO:teuthology.orchestra.run.smithi064.stdout:Building dependency tree...
2019-05-09T16:56:01.658 INFO:teuthology.orchestra.run.smithi064.stdout:Reading state information...
2019-05-09T16:56:01.658 INFO:teuthology.orchestra.run.smithi089.stderr:W: --force-yes is deprecated, use one of the options starting with --allow instead.
2019-05-09T16:56:01.659 INFO:teuthology.orchestra.run.smithi089.stderr:E: Unable to correct problems, you have held broken packages.
```
@batrick @smithfarm

Reply:
Ah, it seems I forgot to adapt these version guards to mimic:

```
Replaces: ceph-common (<< 14.0.0)
Breaks: ceph-common (<< 14.0.0)
```